model/src/main/kotlin/Vulnerability.kt

```diff
@@ -31,6 +34,18 @@ data class Vulnerability(
      */
     val id: String,
 
+    /**
+     * A short description of this [Vulnerability].
```

Comment:
I believe we shouldn't repeat the word "description" here to differentiate from the below. Just say "short summary".